recipe/meta.yaml

```
@@ -93,7 +93,10 @@ outputs:
         - python
       run:
         - python
-        - pyqt >=5.10  # [not osx and not aarch64 and not ppc64le and python_impl == "cpython"]
+        # We avoid the PySide6 dependency on macOS since it uses the MacOSX backend by default
+        - pyside6 >=6.7.2,!=6.7.2=*_0  # [python_impl == "cpython" and win]
+        - pyside6 >=6.7.2,!=6.7.2=*_0  # [python_impl == "cpython" and linux64]
+        - pyside6 >=6.7.2,!=6.7.2=*_0  # [python_impl == "cpython" and aarch64]
```

@hmaarrfk does this look right to you?

---

I guess my syntax knowledge is bad.
```
 pyside6 >=6.7.2,!=6.7.2 *_0 does not exist (perhaps a typo or a missing channel).
```

```
  run_constrained:
     - pyside6 != 6.7.2=*_0
```

---

I agree that putting this in the run-constraint makes sense. Also, I don't see an explanation for the skip on aarch yet. So IMO it should be one line, plus the run-constraint (on all architectures)
```suggestion
        # osx has its own backend; pyside6 not available on ppc
        - pyside6 >=6.7.2  # [python_impl == "cpython" and not (osx or ppc64le)]
```